### PR TITLE
fix: also pass nnodes through the message

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: src/common/slurm_protocol_pack.c [hunk 1]
```diff
@@ -2561,6 +2562,7 @@ _pack_kill_job_msg(kill_job_msg_t * msg, buf_t *buffer, uint16_t protocol_versio
 					     protocol_version);
 		packstr_array(msg->spank_job_env, msg->spank_job_env_size,
 			      buffer);
+    pack32(msg->nnodes, buffer);
 		pack16_array(msg->job_node_cpus, msg->nnodes, buffer);
```
Comment (stdweird) [inline src/common/slurm_protocol_pack.c:2566]: what do you do here with the nnodes?

Comment (stdweird) [reply src/common/slurm_protocol_pack.c:2566]: ah, nnodes here is the size of the array?

**Comment by wdpypere** [comment]
diff looks weird for spacing

**Comment by itkovian** [comment]
@wdpypere Because schedmd uses tabs. I have no tab key :p

**Comment by wdpypere** [comment]
well, best to keep it in line with the rest of the file?

**Comment by stdweird** [comment]
@itkovian i merged this already, since you seem to be making good progress already

**Pull request merged**